### ISSUE TITLE
[no-repetitive-locators] false positive if used in a function

### ISSUE THREAD

**Issue by jrencz** [alecxe/eslint-plugin-protractor#73]
```js
class Helper {
  constructor(selector) {
    this.element = element(by.css(this.selector));
  }

  /**
   * @param {Locator|string} locatorOrSelector
   * @returns {ElementFinder}
   */
  getChild(locatorOrSelector) {
    return this.element.$(locatorOrSelector);
  }

  /**
   * @param {Locator|string} locatorOrSelector
   * @returns {ElementArrayFinder}
   */
  getChildren(locatorOrSelector) {
    return this.element.$$(locatorOrSelector);
  }
}
```

I get false positive in `getChildren`. I checked: in that case `locator` returned by `getLocator` helper is `{ by: 'css', value: undefined }`

**Comment by alecxe** [comment]
@jrencz yup, looks like a bug to me. 

I've also got several places where I had to put it into exceptions (via `eslint-disable`) for some "valid" reasons. But your case is a special case that the rule need not to report about.

Thanks!

**Comment by alecxe** [comment]
Should be fixed by 21a388483a3057a0af300b14c8a1bd2b85ff8e6b. Thanks for the report!

**Comment by jrencz** [comment]
great, than you! May I ask you to ping me here once this is released?

**Comment by alecxe** [comment]
@jrencz sure, it's already there, published to npm automatically.
